3-platform/warehouse/spec/src/main/java/com/_6core/platform/warehousespec/rest/v1/dto/product/ProductSearchResponseCard.java

```diff
@@ -0,0 +1,6 @@
+package com._6core.platform.warehousespec.rest.v1.dto.product;
+
+import java.math.BigInteger;
+
+public record ProductSearchResponseCard(
```

Comment:
1. `Response` should be in the end according to our naming style. 
2. Card is about a flat stiff usually small and rectangular piece of material (such as paper, cardboard, or plastic) usually bearing information, e.g. Credit Card.

Reply:
@sokimaaa, I think about this response object as 'product card' - UI element that shows short info about a product.
![image](https://github.com/sokimaaa/6Core/assets/112807544/1d0db069-b8cb-4e2c-ab25-e89d2e188530)